views/partials/head.ejs

```diff
@@ -13,19 +13,24 @@
   <meta name="twitter:title" content="<%= req.pageName %> - TAMU Datathon">
   <meta name="twitter:description" content="">
 
+  <!-- Specify preconnect hosts -->
+  <link rel="preconnect" href="https://unpkg.com" />
+  <link rel="preconnect" href="https://kit.fontawesome.com" />
+  <link rel="preconnect" href="https://www.googletagmanager.com" />
+  <link rel="preconnect" href="https://www.google-analytics.com" />
```

Comment:
I think this has to be:
```suggestion
  <link rel="preconnect" href="https://www.googletagmanager.com" />
```

Reply:
I think the google tag manager makes a call to google-analytics. The lighthouse report specifies `google-analytics.com` when the preconnect for it isn't there